src/library_sdl.js

```diff
@@ -2893,7 +2903,7 @@ var LibrarySDL = {
       if (channelInfo.audio === this || channelInfo.audio.webAudioNode === this) { 
         channelInfo.audio.paused = true; channelInfo.audio = null; 
       }
-      if (SDL.channelFinished)  {{{ makeDynCall('vi', 'SDL.channelFinished') }}}(channel);
+      if (SDL.channelFinished) {{{ makeDynCall('vi', 'SDL.channelFinished') }}}(channel);
```

Comment:
Braces for this too or does the preprocessor not like that?

Reply:
We do still allow single line if statements and they don't use braces in that case.